nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -2,9 +2,6 @@
 
 from nodewatcher.core.monitor import processors as monitor_processors
 from nodewatcher.modules.monitor.sources.http import processors as http_processors
-
-from django_datastream import datastream
```

Comment:
Not sure if this is the change we were talking. Why this? Topology processor has it. You should be removing that try except statement.